tests/terratest/functions/cleanup/cleanupConfigTF.go

```diff
@@ -0,0 +1,42 @@
+package functions
+
+import (
+	"os"
+)
+
+func CleanupConfigTF() error {
+	path := "../../../../terratest/modules/cluster"
+	file, err := os.Create(path + "/main.tf")
+
+	if err != nil {
+		return err
+	}
+
+	defer file.Close()
+
+	_, err = file.WriteString("// Leave blank - main.tf will be set during testing")
+
```

Comment:
nit; can we remove the space here? Usually, we don't have space with the error check in the Go framework, unless it's large blob of code.